src/main/java/org/wso2/carbon/module/swiftiso20022/mt/models/fields/Field20.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ * <p>
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.module.swiftiso20022.mt.models.fields;
+
+import org.wso2.carbon.module.swiftiso20022.constants.ConnectorConstants;
+import org.wso2.carbon.module.swiftiso20022.constants.MT103Constants;
+import org.wso2.carbon.module.swiftiso20022.constants.MTParserConstants;
+import org.wso2.carbon.module.swiftiso20022.exceptions.MTMessageParsingException;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+
+/**
+ * Model for references in Text Block (Block 04).
+ * <p>
+ * example: :20:Ref254
+ *
+ * @see <a href="https://www2.swift.com/knowledgecentre/publications/usgf_20230720/2.0?topic=idx_fld_tag_20.htm">
+ * Field 20</a>
+ */
+public class Field20 {
+
+    public static final String TAG = "20";
+
+    /**
+     * List of options with current implementation.
+     */
+    private static final List<Character> OPTIONS = Arrays.asList(ConnectorConstants.NO_LETTER_OPTION);
+    public char option;
+
+    // example: Ref254
+    private String value;
+
+    /**
+     * Constructor to initialize all attributes.
+     *
+     * @param option Single character
+     * @param value String with characters from set x.
+     */
+    public Field20(char option, String value) {
+        this.option = option;
+        this.value = value;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    public void setValue(String value) {
+        this.value = value;
+    }
+
+    /**
+     * Method to parse and get Field20 object.
+     * Current implementation -> No_letter
+     *
+     * @param field20String String containing value of 20 field in Text Block
+     * @param option single character option of the field20String
+     * @return An instance of this model.
+     * @throws MTMessageParsingException if the value is invalid
+     */
+    public static Field20 parse(String field20String, char option) throws MTMessageParsingException {
+
+        if (!OPTIONS.contains(option)) {
+            throw new MTMessageParsingException(String.format(MTParserConstants.INVALID_OPTION_FOR_FIELD, option,
+                    ConnectorConstants.FIELD_20));
+        }
+
+        Matcher field20Matcher = MTParserConstants.FIELD_20_REGEX_PATTERN.matcher(field20String);
+
+        if (field20Matcher.matches()) {
+
```

Comment:
is this new line needed?

Reply:
Removed in commit 7b56b383e3127da22c1b7305f9f84c495346f0f3.